collect_app/src/main/res/layout/geopoint_layout.xml

```diff
@@ -57,29 +57,32 @@
 
 		<Button
 			android:id="@+id/reload_location"
-			android:drawableRight="@drawable/ic_add_location_black_36dp"
+			android:drawableEnd="@drawable/ic_add_location_black_36dp"
 			android:padding="15dp"
 			android:layout_margin="5dp"
 			android:layout_width="wrap_content"
 			android:layout_height="match_parent"
-			android:layout_weight="1" />
+			android:layout_weight="1"
+			android:drawableRight="@drawable/ic_add_location_black_36dp" />
```

Comment:
Only one small thing. Please move this line below or above "android:drawableRight" to keep the order like in other cases and everything else is looks ok.